app/_kong_plugins/solace-log/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Solace Log'
+name: 'Solace Log'
+
+content_type: plugin
+
+publisher: kong-inc
+tier: enterprise
+description: 'Publish request and response logs to a Solace topic'
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+min_version:
+    gateway: '3.12'
+
+topologies:
+  on_prem:
+    - hybrid
+    - db-less
+    - traditional
+  konnect_deployments:
+    - hybrid
+    - cloud-gateways
+    - serverless
+
+tags:
+  - logging
+  - events
+  - solace
+
+search_aliases:
+  - solace-log
+  - events
+  - event-gateway
+
+icon: solace-log.png
+
+categories:
+   - logging
+
+related_resources:
+  - text: Solace Upstream plugin
+    url: /plugins/solace-upstream/
+  - text: Solace Consume plugin
+    url: /plugins/solace-consume/
+  - text: Event Gateway
+    url: /event-gateway/
+---
+
+Publish request and response logs to a [Solace](https://solace.com/) topic.
+For more information, see [Understanding Solace topics](https://docs.solace.com/Get-Started/what-are-topics.htm).
+
+Kong also provides Solace plugins for request and response transformations:
+* [Solace Upstream](/plugins/solace-upstream/)
+* [Solace Consume](/plugins/solace-consume/)
+
+## Log format
+
+{% include /plugins/logging/log-format.md %}
+
+## Implementation details
+
+TBA
```

Comment:
```suggestion
This plugin leverages the [log PDK](/gateway/pdk/reference/kong.log/) to collect and [customize](#custom-fields-by-lua) log fields.

The prepared log message is sent to Solace broker via the official [Solace C API](https://docs.solace.com/API/Messaging-APIs/C-API/c-api-home.htm). The sending job is executed in a background timer context, such that it would not block the client requests.

If the [custom Lua code](#custom-fields-by-lua) associated with the log fields fails to execute, the relavent fields remain untouched.
```